spec/controllers/admin/contributions_controller_spec.rb

```diff
@@ -66,6 +66,21 @@
     end
   end
 
+  describe 'PUT request_refund' do
+    let(:contribution) { create(:confirmed_contribution) }
+    subject { payment.pending_refund? }
+
+    before do
+      allow(payment).to receive(:direct_refund).and_return(true)
+      put :request_refund, id: payment.id, locale: :pt
+    end
+
+    it do
+      payment.reload
+      is_expected.to eq(true)
```

Comment:
Using the response as subject this test would become explicit and we should use a description in the it :D